lusSTR/annot.py

```diff
@@ -762,21 +763,23 @@ def flank_3(full_seq, back, locus, n):
 def main(args):
     cannot_split = [
         "D19S433", "D6S1043", "TH01", "D21S11", "D1S1656", "D7S820", "D5S818", "D12S391",
-        "D9S1122", "D1S1656", "PentaE"
+        "D9S1122", "PentaE"
     ]
     must_split = ["D13S317", "D18S51"]
 
-    sample_file = open(args.input, "r")
-    data = csv.reader(sample_file)
-    next(data)
-    for row in data:
-        locus = row[0]
-        reads = row[1]
-        sequence = row[2]
-        sampleid = row[3]
+    data = pd.read_csv(args.input)
+    final_output = pd.DataFrame()
+    final_flank_output = pd.DataFrame()
+    list_of_lists = []
+    flanks_list = []
+    for i in range(len(data)):
+        locus = data.iloc[i, 0]
+        reads = data.iloc[i, 1]
+        sequence = data.iloc[i, 2]
+        sampleid = data.iloc[i, 3]
```

Comment:
Great!

Reply:
So I should say that one of the reasons I didn't use column names is so if someone provides their own formatted file as input but the column names aren't exactly correct ('locus' instead of 'Locus', etc.), it won't err out the script.

Reply:
Event still: is there a good reason *not* to require that their column names are correct? (Maybe add column names validation check when you first load the dataframe?)

In any case, you should be able to access columns by position on a row object (might even be `.iloc`), so you could at least `iterrows`.

Reply:
Personal preference? I get annoyed when I get errors because of column names being slightly incorrect. :)

Reply:
I did change to `iterrows` at least but I'm sticking to my .iloc :)